dev-python/email-validator/email-validator-1.1.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
Please be more concise and group related vars together, i.e. DISTUTILS and PYTHON.